test/t8_schemes/t8_gtest_nca.cxx

```diff
@@ -170,50 +161,72 @@ t8_recursive_nca_check(t8_element_t *check_nca, t8_element_t *desc_a,
     int level_b = ts->t8_element_level(parent_b);
     int num_children_a, num_children_b;
     int i, j;
-    /* If one parent has reached the maximal level, the test returns*/
+    /* If one parent has reached the maximal level, the test returns */
     if(level_a == max_lvl || level_b == max_lvl){
         return;
     }
     num_children_a = ts->t8_element_num_children(parent_a);
     num_children_b = ts->t8_element_num_children(parent_b);
+    
     /* Iterate over all children of parent_a */
     for(i = 0; i < num_children_a; i++){
         ts->t8_element_child(parent_a, i, desc_a);
-        /* Iterate over all children of parent_b*/
+        /* Iterate over all children of parent_b */
         for(j = 0; j < num_children_b; j++){
             ts->t8_element_child(parent_b, j, desc_b);
-
-            /*Compute the nca and check if it is equal to correct_nca */
             ts->t8_element_nca(desc_a, desc_b, check);
+            
             if(ts->t8_element_compare(check_nca, check)){
```

Comment:
Just for increasing code clarity. I would make the check more explicit by writing
`ts->t8_element_compare(check_nca, check) != 0`.